tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -32,3 +26,21 @@ jobs:
         inputs:
           artifactName: 'wheel_$(Agent.OS)_$(Agent.JobName)'
           targetPath: 'flink-python/dist'
+
+  - job: build_wheels_on_macos
+    pool:
+      vmImage: 'macOS-latest'
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.9'
```

Comment:
Does it mean that since 1.16, users who want to run pyflink in m1 mac should at least install python3.9?

Reply:
My previous understanding is wrong as it is used to specify the py version of cibuildwheel's env.
Now, the PR LGTM.